README.md

```diff
@@ -175,7 +175,7 @@ $response = $matej->request()
     ->addItemProperty(ItemProperty::create('item-id', ['valid_from' => time(), 'title' => 'Title']))
     ->addItemProperties([/* array of ItemProperty objects */])
     // Merge user
-    ->addUserMerge(UserMerge::mergeInto('target-user-id', 'source-user-id'))
+    ->addUserMerge(UserMerge::mergeInto('target-user-id', 'source-user-id'), time())
```

Comment:
I don't think this case with `time()` as the value should be in the example - this replicates the default behavior  (`time()` is used if no other value is provided), so this exact implementation is not useful. Maybe put there some other value and mention the current timestamp is used if no other value is given?

Reply:
I've added whole section about merging. I don't go over details as they are described in matej documentation.